packages/react-vapor/src/components/table-hoc/tests/TableSelectors.spec.ts

```diff
@@ -3,6 +3,44 @@ import {TableSelectors} from '../TableSelectors';
 import {ITableHOCCompositeState, TableHOCUtils} from '../utils/TableHOCUtils';
 
 describe('TableSelectors', () => {
+    describe('getIsEmpty', () => {
+        describe('when the "isServer" prop is true', () => {
+            it('returns true if the "data" prop is undefined', () => {
+                expect(TableSelectors.getIsEmpty({}, {id: '🌶', isServer: true, data: undefined})).toBe(true);
+            });
+
+            it('returns true if the "data" prop is null', () => {
+                expect(TableSelectors.getIsEmpty({}, {id: '🌶', isServer: true, data: null})).toBe(true);
+            });
+
+            it('returns true if the "data" prop is an empty array', () => {
+                expect(TableSelectors.getIsEmpty({}, {id: '🌶', isServer: true, data: []})).toBe(true);
+            });
+
+            it('returns false if the "data" prop is a non-empty array', () => {
+                expect(TableSelectors.getIsEmpty({}, {id: '🌶', isServer: true, data: [1, 2, 3]})).toBe(false);
+            });
+        });
+
+        describe('when the "isServer" prop is false', () => {
+            it('returns true if the "data" prop is undefined', () => {
+                expect(TableSelectors.getIsEmpty({}, {id: '🌶', isServer: false, data: undefined})).toBe(true);
+            });
+
+            it('returns true if the "data" prop is an empty array', () => {
+                expect(TableSelectors.getIsEmpty({}, {id: '🌶', isServer: false, data: []})).toBe(true);
+            });
+
+            it('returns false if the "data" prop is null', () => {
+                expect(TableSelectors.getIsEmpty({}, {id: '🌶', isServer: false, data: null})).toBe(false);
+            });
+
+            it('returns false if the "data" prop is a non-empty array', () => {
+                expect(TableSelectors.getIsEmpty({}, {id: '🌶', isServer: false, data: [1, 2, 3]})).toBe(false);
+            });
+        });
+    });
+
```

Comment:
spicy tests

Reply:
that's how I know which tests were made by Gael :joy:  :hot_pepper: